config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml

```diff
@@ -355,6 +355,111 @@ periodics:
         resources:
           requests:
             cpu: 2000m
+- interval: 6h
+  name: ci-kubernetes-csi-1-14-on-kubernetes-1-13
```

Comment:
I'm unsure if this configuration is valid.  A driver deployment for 1.14 could have enabled features that will only work on Kubernetes 1.14 (like the new Leases leader election type).

Reply:
True, it may or may not be compatible. But we know that it will work for the 1.14 deployment, therefore this combination is valid and worth testing. For example, https://kubernetes-csi.github.io/docs/external-attacher.html says that the minimum Kubernetes version is 1.13. This job covers that promise.

Reply:
Once we add leader election, topology or any of the CSI driver features, then it will no longer be the case.  

I'm thinking in the future if we want to better automate setting up these jobs, then it's simpler to have rules that work across all scenarios

Reply:
I consider the kubernetes-1.14 deployment mostly frozen at this point. It enables a certain set of features and those happen to be compatible with 1.13 (when ignoring alpha). IMHO it is useful to keep it this way, to increase test coverage of the stable images also with Kubernetes 1.13.

When we added leader election, I would do it in a separate deployment because we need cover both scenarios (with and without leader election). That new deployment then cannot run on 1.13.

Reply:
Having said that, if you don't care about testing this particular aspect, then I won't argue further in favor of keeping it. It's not a combination that I personally use.

Reply:
I mainly don't want this to be misinterpreted as a general statement that we support running these sidecars against 1.13 in all scenarios.  It's very dependent per driver and what features they use.

If we eventually add these 1.14+ features to the hostpath-driver, our deployment management and version skew testing will become very complex if we want to be able to test with all theoretical combinations of new/old features and versions.